### ISSUE TITLE
DynamoDB tags are not returned with ListTagsOfResource

### ISSUE THREAD

**Issue by sherzzo** [localstack/localstack#2191]
# Type of request:

[x ] bug report

# Detailed description

DynamoDB tags are not returned with `ListTagsOfResource` call. Here is my attempt with AWS CLI:

CLI playload
```json
{
    "AttributeDefinitions": [
        {
            "AttributeName": "id",
            "AttributeType": "S"
        }
    ],
    "TableName": "bar",
    "KeySchema": [
        {
            "AttributeName": "id",
            "KeyType": "HASH"
        }
    ],
    "ProvisionedThroughput": {
        "ReadCapacityUnits": 5,
        "WriteCapacityUnits": 5
    },
    "Tags": [
        {
            "Key": "Test",
            "Value": "true"
        }
    ]
}
```

Commands

```bash
$ aws dynamodb --endpoint-url http://localhost:4569 create-table \
--cli-input-json file://db.json
{
    "TableDescription": {
        "AttributeDefinitions": [
            {
                "AttributeName": "id",
                "AttributeType": "S"
            }
        ],
        "TableName": "bar",
        "KeySchema": [
            {
                "AttributeName": "id",
                "KeyType": "HASH"
            }
        ],
        "TableStatus": "ACTIVE",
        "CreationDateTime": 1585073349.731,
        "ProvisionedThroughput": {
            "LastIncreaseDateTime": 0.0,
            "LastDecreaseDateTime": 0.0,
            "NumberOfDecreasesToday": 0,
            "ReadCapacityUnits": 5,
            "WriteCapacityUnits": 5
        },
        "TableSizeBytes": 0,
        "ItemCount": 0,
        "TableArn": "arn:aws:dynamodb:us-east-1:000000000000:table/bar",
        "BillingModeSummary": {
            "BillingMode": "PROVISIONED",
            "LastUpdateToPayPerRequestDateTime": 0.0
        }
    }
}
```

Then I tried listing table's tags

```bash
$ aws dynamodb --endpoint-url http://localhost:4569 list-tags-of-resource \
--resource-arn arn:aws:dynamodb:us-east-1:000000000000:table/bar
{
    "Tags": []
}
```
...and got empty list. Tried to add one more tag

```bash
$ aws dynamodb --endpoint-url http://localhost:4569 tag-resource \
--resource-arn arn:aws:dynamodb:us-east-1:000000000000:table/bar \
--tags Key=Name,Value=Bar

$ $ aws dynamodb --endpoint-url http://localhost:4569 list-tags-of-resource \
--resource-arn arn:aws:dynamodb:us-east-1:000000000000:table/bar
{
    "Tags": []
}
```
...still got an empty list.

## Expected behavior

List of tags

```json
    "Tags": [
        {
            "Key": "Test",
            "Value": "true"
        }
    ]
}
```

...

## Actual behavior

Always getting empty list.
...

# Steps to reproduce

## Command used to start LocalStack

```bash
TMPDIR=/private$TMPDIR docker-compose up -d
```
...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

Create CLI skeleton
```bash
cat <<EOL>> db.json
{
    "AttributeDefinitions": [
        {
            "AttributeName": "id",
            "AttributeType": "S"
        }
    ],
    "TableName": "bar",
    "KeySchema": [
        {
            "AttributeName": "id",
            "KeyType": "HASH"
        }
    ],
    "ProvisionedThroughput": {
        "ReadCapacityUnits": 5,
        "WriteCapacityUnits": 5
    },
    "Tags": [
        {
            "Key": "Test",
            "Value": "true"
        }
    ]
}
EOL
```

Create and list

```bash
$ awslocal dynamodb create-table --cli-input-json file://db.json
$ awslocal dynamodb list-tags-of-resource \ 
--resource-arn arn:aws:dynamodb:us-east-1:000000000000:table/bar
```

**Comment by sherzzo** [comment]
This feels more like a feature request, as I see that response of `ListTagsOfResource ` was stubbed with empty JSON.

https://github.com/localstack/localstack/blob/c248c0e8073313cbc68c4bc73bac95c5366ad95e/localstack/services/dynamodb/dynamodb_listener.py#L142

**Comment by whummer** [comment]
Thanks for the detailed report @sherzzo . This should be fixed in #2219 - can you please give it a try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks!

**Comment by sherzzo** [comment]
Thanks @whummer , looks good on my end!
